accelerometer/summariseEpoch.py

```diff
@@ -359,13 +360,10 @@ def calculateECDF(e, inputCol, summary, useRecommendedImputation):
     :return: Write dict <summary> keys '<inputCol>-ecdf-<level...>mg'
     :rtype: void
     """
-
-    ecdf1, step = np.linspace(1, 20, 20, retstep=True)  # 1mg bins from 1-20mg
-    ecdf2, step = np.linspace(25, 100, 16, retstep=True)  # 5mg bins from 25-100mg
-    ecdf3, step = np.linspace(125, 500, 16, retstep=True)  # 25mg bins from 125-500mg
-    ecdf4, step = np.linspace(600, 2000, 15, retstep=True)  # 100mg bins from 500-2000mg
-    ecdfXVals = np.concatenate([ecdf1, ecdf2, ecdf3, ecdf4])
-
+    filename = intensityDistributionVals[0]
+    valsFile = open(filename, "r")
+    ecdfXVals = [float(number) for number in valsFile]
```

Comment:
Hmm you normally have to make sure you closed a file after you're done with it to release resources (`valsFile.close()` after done). In Python there's a convenient way to ensure this with [with](https://stackoverflow.com/questions/1369526/what-is-the-python-keyword-with-used-for) statements:

```python
with open(filename, 'r') as valsFile:
    ecdfXVals = [float(number) for number in valsFile]
```
Once outside the `with`, it will automatically close `valsFile` for you.